src/test/java/HomeWork19.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import java.time.Duration;
+
+public class HomeWork19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        /*openLoginUrl();*/
+        provideEmail("vera1077@gmail.com");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        selectPlaylist();
+        clickDeletePlaylist();
+    }
+    public void selectPlaylist() throws InterruptedException {
+        WebElement selectPlaylistToDelete = driver.findElement(By.xpath("//*[@id=\"playlists\"]/ul/li[3]/a"));
+        Thread.sleep(2000);
+        selectPlaylistToDelete.click();
+    }
+
+    public static void clickDeletePlaylist() throws InterruptedException{
+        WebElement deleteBtn = driver.findElement(By.xpath("//button[@class='del btn-delete-playlist']"));
+        deleteBtn.click();
+        Thread.sleep(2000);
+        WebElement deleteConfirm = driver.findElement(By.xpath("//button[@class='ok']"));
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/227135248-a8336cb6-9e1a-46fe-ad9a-22145e99589b.png)